### PR TITLE
Update Covid banner text

### PR DESCRIPTION
**Pull request by patch0**
See CoderDojo/cp-translations#123 for the required changes.

### CODE REVIEW

**Comment by DanielBrierton** [comment]
Deploy preview for *cdf-website* ready!

Built with commit d655baf7c8febe04f42e62b931e26f3c83f93a5b

https://deploy-preview-62--cdf-website.netlify.app

**Comment by DanielBrierton** [comment]
Deploy preview for *cdf-website* ready!

Built with commit 062091e913a78b4d9e28349504845b2dc3050431

https://deploy-preview-62--cdf-website.netlify.app

**Comment by DanielBrierton** [comment]
Deploy preview for *cdf-website* ready!

Built with commit bc8cb8d9347807e2e72332903f29acd9b4fab1c5

https://deploy-preview-62--cdf-website.netlify.app

File: components/Banner.vue [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
+      </p>
+      <p>
+        {{ $t('If you are ready to hold in-person club events soon, please') }}
+        <a class="cd-banner-link" href="https://zen.coderdojo.com/dashboard/start-dojo">{{ $t('register your details') }}</a>
+        {{ $t('and we will be in touch.') }}
+      </p>
+      <p>
+        {{ $t('If you’re not able to start your Dojo in person yet, there are many other ways that you can get involved in the meantime.') }}
+        {{ $t('Check out the exciting opportunities for young people, parents, volunteers, and educators to learn and get creative with tech through') }}
+        <a class="cd-banner-link" target="_blank" href="https://www.raspberrypi.org/learn/?utm_source=coderdojo&utm_medium=covid-banner&utm_campaign=dmah">{{ $t('Digital Making at Home') }}</a>
```
Comment (glenpike) [inline components/Banner.vue:17]: escape the ampersands like the other one?

**Pull request merged**